### DISCUSSION TITLE
Sheep detection - detects bighorns

### DISCUSSION THREAD

**Discussion by summitbri** [blakeblackshear/frigate#1000] [Show and tell]
Just a shout out to @blakeblackshear, love this project.  Our use case is more around detecting wildlife than bad actors.  Check it out: 

https://www.youtube.com/watch?v=u72VofspSag

HA automates our window blinds based on sheep detection and they slowly walk away from the windows :)

**Comment by mattheys** [comment]
Thats really cool.
